tests/conftest.py

```diff
@@ -251,15 +258,23 @@ def pytest_collection_modifyitems(config, items):
     if config.getoption("--docs-tests"):
         # --docs-tests given in cli: do not skip documentation integration tests
         return
+    if config.getoption("--cloud"):
```

Comment:
Should this `if` block be here? It looks like it early exits if any of these flags are present so passing any of them is equivalent to passing all of them.